src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Comparator;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+/**
+ * Sort all persons in address book by the parameter.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": sort all persons by name or appointment date.\n "
+            + "Parameters: KEYWORD [PARAMETER name/ or appt/]\n"
+            + "Example: " + COMMAND_WORD + " name/";
```

Comment:
```suggestion
            + "Example: " + COMMAND_WORD + " n/";
```

in accordance to the comment above, should you decide to alter

Reply:
Thanks for noticing the mistakes! fixed!